ecosystem/web-wallet/src/utils/account.ts

```diff
@@ -26,3 +26,29 @@ export function createNewAccount (): Result<AptosAccount, Error> {
   // todo: make request to create account on chain
   return ok(account)
 }
+
+export function getLocalStorageState (): LocalStorageState | null {
+  if (typeof window === 'undefined') {
+    return null
+  }
+  try {
+    // Get from local storage by key
+    const item = window.localStorage.getItem(walletStateLocalStorageKey)
+    return item ? JSON.parse(item) : null
+  } catch (_err) {
+    return null
```

Comment:
Why silence the error? I think it should be an exception if we're unable to parse from localstorage.